models/node_helpers.go

```diff
@@ -80,9 +80,15 @@ func checkUniqueNodeInstanceRegion(q *reform.Querier, instance, region string) e
 	}
 }
 
-// FindAllNodes returns all Nodes.
-func FindAllNodes(q *reform.Querier) ([]*Node, error) {
-	structs, err := q.SelectAllFrom(NodeTable, "ORDER BY node_id")
+// FindNodes returns Nodes by filters.
+func FindNodes(q *reform.Querier, nodeType *NodeType) ([]*Node, error) {
```

Comment:
NodeFilters instead of NodeType?

Reply:
I did that way, but for now we have only one filter and it doesn't look like we will have other ones. So decided to use NodeType instead of NodeFilters

Reply:
But then NodeFilters is not used anywhere?